sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -1020,6 +1020,45 @@ class CodegenContext extends Logging {
     genCodes
   }
 
+  /**
+   * Defines an independent function for a given expression and returns a caller-side code
+   * for the function as `ExprCode`.
+   */
+  def defineIndependentFunction(
```

Comment:
What does `Independent` stand for?

Reply:
Ur, I made a mistake trying to write `Individual`.. I'll update soon.